### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by why-not-try-calmer** [openSUSE/openSUSE-docs-revamped-temp#35]
`WARNING  -  Documentation file 'yast_installer.md' contains a link to 'backing_up.md' which is not found in the documentation files.`

Could you please take a look @apinter  ;) ?

**Comment by apinter** [comment]
Wondering what was the intention behind that :thinking:

**Comment by why-not-try-calmer** [comment]
> Wondering what was the intention behind that

Hhehe sorry I am asking if you'd be willing to take a look at the links from `yast_installer.md`, because we have one broken link apparently.

**Comment by apinter** [comment]
Sure, but I meant behind putting that link there. Feels like that there was a reason for that heehe

**Comment by why-not-try-calmer** [comment]
Aw my bad I realize I myself probably put this link there. It's been a while 👯 ... 

Do you have recommendations for backup tools to recommend the user?

**Comment by apinter** [comment]
It's funny you ask cause I'm building to a disaster recovery and backup strategy at $Dayjob right now and would probably have a few things to recommend for backing up. Would be even easier with btrfs actually, but can mention a few tools like `Borg` or `Timeshift` or `DejaDup`. All of which are available in the official repos. That is why I try to figure what is the reason behind the link, cause IMHO writing the doc would be actually a better idea than removing the link.

**Comment by why-not-try-calmer** [comment]
Yes writing the doc was the intention behind my last message. And the question was whether you'd be willing to write it.

**Comment by Ethanol6** [comment]
> It's funny you ask cause I'm building to a disaster recovery and backup strategy at $Dayjob right now and would probably have a few things to recommend for backing up. Would be even easier with btrfs actually, but can mention a few tools like `Borg` or `Timeshift` or `DejaDup`. All of which are available in the official repos. That is why I try to figure what is the reason behind the link, cause IMHO writing the doc would be actually a better idea than removing the link.

Timeshift is not recommended for `/home` directory backups. I can also recommend `restic` and `tarsnap`.
